core-bundle/src/Resources/contao/languages/en/tl_image_size.xlf

```diff
@@ -65,6 +65,12 @@
       <trans-unit id="tl_image_size.skipIfDimensionsMatch.1">
         <source>Note that no optimizations such as meta data stripping or JPEG compression will be applied in this case.</source>
       </trans-unit>
+      <trans-unit id="tl_image_size.lazyLoading.0">
+        <source>Enable lazy loading</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.lazyLoading.1">
+        <source>Adds loading=&amp;quot;lazy&amp;quot; to the &amp;lt;img&amp;gt; tag to defer loading the image until it is scrolled into the viewport.</source>
```

Comment:
Couldn't you also just write `&` instead of `&amp;` here?

Reply:
No. I tried. Doesn’t work with `&lt;img&gt;`.

Reply:
I see. Here too, for example: https://github.com/contao/contao/blob/master/core-bundle/src/Resources/contao/languages/en/explain.xlf#L81

That's curious. What's so special with `&lt;img&gt;`, because it seems to work with all other elements?

Reply:
In your example the output results in an actual `<code>` tag, but here we don’t want an actual `<img>` tag but “&lt;img&gt;” as text instead.